test/functionalTest/cases/2560_custom_notification_json/custom_notification_mqtt_json_replacements_inside.test

```diff
@@ -16,7 +16,7 @@
 # along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
 #
 # For those usages not covered by this license please contact with
-# iot_support at tid dot es
+# iot_support at tid dot es.
```

Comment:
```suggestion
# iot_support at tid dot es
```

That little dot is causing a fail in the compliance check script ;) https://github.com/telefonicaid/fiware-orion/actions/runs/4499061857/jobs/7938064519?pr=4300